Package.swift

```diff
@@ -6,30 +6,31 @@ let package = Package(
     name: "SplunkOtel",
     platforms: [
         .iOS(.v11),
-	.macOS(.v10_13)
+        .macOS(.v10_13)
     ],
     products: [
         .library(name: "SplunkOtel", targets: ["SplunkOtel"])
     ],
     dependencies: [
         .package(name: "opentelemetry-swift", url:"https://github.com/open-telemetry/opentelemetry-swift", from: "1.1.4"),
-	.package(url: "https://github.com/devicekit/DeviceKit.git", from: "4.0.0"),
+        .package(url: "https://github.com/devicekit/DeviceKit.git", from: "4.0.0"),
     ],
     targets: [
         .target(
             name: "SplunkOtel",
             dependencies: [
-		.product(name: "OpenTelemetryApi", package:"opentelemetry-swift"),
-		.product(name: "OpenTelemetrySdk", package:"opentelemetry-swift"),
-		.product(name: "StdoutExporter", package:"opentelemetry-swift"),
-		.product(name: "ZipkinExporter", package:"opentelemetry-swift"),
-		.product(name: "DeviceKit", package: "DeviceKit")
+                .product(name: "OpenTelemetryApi", package:"opentelemetry-swift"),
+                .product(name: "OpenTelemetrySdk", package:"opentelemetry-swift"),
+                .product(name: "StdoutExporter", package:"opentelemetry-swift"),
+                .product(name: "ZipkinExporter", package:"opentelemetry-swift"),
+                .product(name: "DeviceKit", package: "DeviceKit")
             ],
             path: "SplunkRumWorkspace/SplunkRum",
             exclude: [
-		"SplunkRumTests",
-		"SplunkRum/SplunkRum.h",
-		"SplunkRum/Info.plist"
+                "SplunkRumTests",
+                "SplunkRumDiskExportTests",
```

Comment:
I re-indented the whole file. But more importantly, I also added SplunkRumDiskExportTests to the exclude list. This get rid of a package warning that those files were not properly handled.